rust/xaynet-sdk/src/state_machine/phases/update.rs

```diff
@@ -255,7 +233,11 @@ impl Phase<Update> {
             // method
             local_seed_dict: self.state.private.seed_dict.take().unwrap(),
         };
-        self.state.private.message = Some(self.message_encoder(update.into()));
-        Progress::Updated(self.into())
+        let message = self.message_encoder(update.into());
+
+        debug!("going to sending phase");
+        let sending = Sending::from_sum2(message);
```

Comment:
this seems to be a typo (it accidentally does the correct thing internally):
```suggestion
        let sending = Sending::from_update(message);
```

Reply:
fixed.